db/migrate/20190826133124_add_car_id_to_trains.rb

```diff
@@ -0,0 +1,5 @@
+class AddCarIdToTrains < ActiveRecord::Migration[5.2]
+  def change
+    add_belongs_to :trains, :car
```

Comment:
???

Reply:
so it was in the video

Reply:
```class AddRouteIdToTrains < ActiveRecord::Migration[5.2]
  def change
    add_belongs_to :trains, :route
  end
end```

Reply:
Yes route, because train belongs to route!!!

Reply:
Car belongs to train too